test/tools/ossfuzz/config/strict_assembly.dict

```diff
@@ -0,0 +1,118 @@
+" -> "
+" := "
+"("
+")"
+", "
+":"
+" dup1 "
+" dup10 "
+" dup11 "
+" dup12 "
+" dup13 "
+" dup14 "
+" dup15 "
+" dup16 "
+" dup2 "
+" dup3 "
+" dup4 "
+" dup5 "
+" dup6 "
+" dup7 "
+" dup8 "
+" dup9 "
+" pc "
+" stop "
+" swap1 "
+" swap10 "
+" swap11 "
+" swap12 "
+" swap13 "
+" swap14 "
+" swap15 "
+" swap16 "
+" swap2 "
+" swap3 "
+" swap4 "
+" swap5 "
+" swap6 "
+" swap7 "
+" swap8 "
+" swap9 "
+"add("
+"addmod("
+" address "
+"and("
+"balance("
+"blockhash("
+"byte("
+"call("
+"callcode("
+"calldatacopy("
+"calldataload("
+" calldatasize "
+" caller "
```

Comment:
In strict assembly, all opcodes must be used as functions, even if they have no arguments, so I would propose to use `caller()` here, `pc()` above and similar for other opcodes.

Reply:
okay.